packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -945,6 +945,14 @@ const messages = {
     resource: {
       accessDenied: 'We are sorry, but this resource is only available to teachers who are logged in with Feide.',
     },
+    primarySchool: 'Primary School',
+    name: 'Name',
+    mail: 'Mail',
```

Comment:
Bør vel ha samme som på norsk.
```suggestion
    mail: 'E-mail',
```